scripts/verify-no-bundled-agent-runtimes.sh

```
@@ -0,0 +1,76 @@
+  strings -a "$path" 2>/dev/null | grep -Eq '(/\$bunfs/|StandaloneExecutable|Bun v[0-9]+\.[0-9]+\.[0-9]+)'
+}
+
+violations=()
+
+while IFS= read -r -d '' file; do
+  if [ ! -x "$file" ] && [ ! -L "$file" ]; then
+    continue
+  fi
+  name="$(basename "$file")"
+  if ! is_allowed_binary_name "$name"; then
+    violations+=("unexpected executable: ${file#$APP_PATH/}")
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)